build-system/amp.extern.js

```diff
@@ -17,26 +17,32 @@
 /** @externs */
 
 /**
- * Special case for fetchJson.
+ * The "init" argument of the Fetch API. Currently, only "credentials: include"
+ * is implemented.  Note ampCors with explicit false indicates that
+ * __amp_source_origin should not be appended to the URL to allow for
+ * potential caching or response across pages.
+ *
+ * See https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch
+ *
  * Externed for use in the amp-form component when reconstructing
  * the request for SSR and then passed to runtime.
  * @typedef {{
- *   body: (!JsonObject|!FormData|undefined),
+ *   body: (!JsonObject|!FormData|!Object|!Array|undefined|string),
```

Comment:
Revert this change.

Reply:
Doing so breaks check-types run, though Array, I was able to remove. Certain extensions refer to the 'body' field expecting it to be of type object type or string.